initialize/initialize.go

```diff
@@ -129,6 +133,64 @@ func githubSearch(query string) ([]Repo, error) {
 
 	client := github.NewClient(tc)
 
+	opts := &github.SearchOptions{}
```

Comment:
Seems like this is all pretty identical to the `githubRepoSearch` function, can you separate it out?

Reply:
Could you explain a bit more what do you mean by separating it out?

Reply:
I meant separating the shared code into its own function.